README.md

```diff
@@ -133,6 +133,7 @@ from PIL import Image
 
 # Choose GPU instead of CPU in the line below to run the model on Intel integrated or discrete GPU
 pipe = ov_genai.VLMPipeline("./InternVL2-1B", "CPU")
+pipe.start_chat()
 
```

Comment:
BTW, below we have `print(pipe.generate(prompt, image=image_data, max_new_tokens=100))` which returns `VLMDecodedResults`

should we fix it to extract `texts[0]` from this structure?

Reply:
Thanks, I added that to this PR. Apparently VLMDecodedResults has a `.__str__` method that shows the text so it is not immediately noticeable that this generate method does not actually return the text; I agree it's better to be explicit about this in the example.